dbt/include/global_project/macros/materializations/snapshots/strategies.sql

```diff
@@ -133,11 +135,11 @@
 {%- endmacro %}
 
 
-{% macro snapshot_check_strategy(node, snapshotted_rel, current_rel, config, target_exists) %}
-    {% set check_cols_config = config['check_cols'] %}
-    {% set primary_key = config['unique_key'] %}
-    {% set invalidate_hard_deletes = config.get('invalidate_hard_deletes', false) %}
-    {% set updated_at = config.get('updated_at', snapshot_get_time()) %}
+{% macro snapshot_check_strategy(node, snapshotted_rel, current_rel, model_config, target_exists) %}
```

Comment:
It looks liked the kwarg was updated but the original name is still referenced below. Is that intentional?

Reply:
Do you mean the "model_config"?  Yeah, I left the param there for cases where somebody might have been using it, but renamed it to make it clear that it's not the same config that's used below.

Reply:
I can foresee a future in which the significance of this move is lost on whoever's reading this. Mind adding a comment that we left the model config to keep same interface or something to that effect?

Reply:
I've added some comments where it's passed in and in the functions.